### PR TITLE
Image related updates

### PR DESCRIPTION
**Pull request by ncdc**

### CODE REVIEW

File: architecture/core_concepts/builds_and_image_streams.adoc [hunk 1]
```diff
@@ -296,15 +289,24 @@ for the tag "2.0":
   }
 }
 ----
+
+You can also do this using the `oc tag` command:
+
+`oc tag ruby:latest ruby:2.0`
+
```
Comment (adellape) [inline architecture/core_concepts/builds_and_image_streams.adoc]: I think the above lines should be outside the ==== block?

File: architecture/core_concepts/builds_and_image_streams.adoc [hunk 2]
```diff
@@ -296,15 +289,24 @@ for the tag "2.0":
...
 ====
 
 === Image Stream Mappings
-When the integrated OpenShift docker registry receives a new image, a
-notification is sent to OpenShift via the `*ImageStreamMapping*` route. The
-registry informs OpenShift of the image's namespace, name, tag, and docker
-metadata. OpenShift uses this information to create a new image and to update
-the appropriate image stream with the mapping between the image's tag and the
-image itself.
+When the integrated OpenShift Docker Registry receives a new image, it creates
```
Comment (adellape) [inline architecture/core_concepts/builds_and_image_streams.adoc]: I notice you usually initcap "Docker Registry". Preference, or based on another reason?

File: architecture/core_concepts/builds_and_image_streams.adoc [hunk 3]
```diff
@@ -296,15 +289,24 @@ for the tag "2.0":
...
+and sends an `*ImageStreamMapping*` to OpenShift. This informs OpenShift of the
+image's namespace, name, tag, and Docker metadata. OpenShift uses this
+information to create a new image (if it doesn't already exist) and to tag the
```
Comment (adellape) [inline architecture/core_concepts/builds_and_image_streams.adoc]: s/doesn't/does not/

File: architecture/core_concepts/builds_and_image_streams.adoc (hunk 2 above)
Comment (ncdc) [inline architecture/core_concepts/builds_and_image_streams.adoc]: I assume Docker should be capitalized at least. I'm ok not capitalizing registry, but "OpenShift Docker registry" seems a bit... weird, maybe?

**Comment by adellape** [comment]
@ncdc Minor nits inline, otherwise LGTM.

cc @sosiouxme

**Comment by ncdc** [comment]
@adellape comments addressed

**Comment by adellape** [comment]
@ncdc Sorry, can you rebase this? I think it would undo some fixes I got in since the 550 PR.

**Comment by ncdc** [comment]
@adellape rebased. Can you double-check to make sure it's ok w.r.t your post-550 fixes?

**Comment by adellape** [comment]
@ncdc LGTM, thanks!

**Pull request merged**